Server/Server.Test/Utilities/TestUtility.hpp

```diff
@@ -115,11 +115,8 @@ inline Message& makeMessage(Message& message, MessageType messageType, MessageBo
                 message.mBody           = std::make_any<RegistrationInfo>(registrationInfo);
                 break;
             }
-            else
-            {
-                RegistrationInfo registrationInfo(mesgFiller.getRegistrationInfo());
-                message.mBody = std::make_any<RegistrationInfo>(registrationInfo);
-            }
+            RegistrationInfo registrationInfo(mesgFiller.getRegistrationInfo());
+            message.mBody = std::make_any<RegistrationInfo>(registrationInfo);
```

Comment:
It's the same logically.
But I would prefer the next realization:
        {
            if (messageBody == MessageBody::InvalidBody)
            {
                RegistrationInfo registrationInfo;
                registrationInfo._login = "";
                message.mBody           = std::make_any<RegistrationInfo>(registrationInfo);
             }
            else
            {
                RegistrationInfo registrationInfo(mesgFiller.getRegistrationInfo());
                message.mBody = std::make_any<RegistrationInfo>(registrationInfo);
            }
            break;
        }
Because thit way is more readable, understandable and unambiguous.
This is just a comment.

Reply:
I understand you perfectly, but there are too many if else blocks, as it reads much more complicated to me. There is a cognitive complexity from too many blocks:(